portfolio/src/main/webapp/styles/style.css

```diff
@@ -1,35 +1,36 @@
 /* General Elements */
 body {
-  color:rgb(160, 160, 160);
-  background-color: #02003C;
+  color:white;
+  background-color: #41485B;
 }
 
 img {
   height: 10%;
   width: 10%;
 }
 
+h1 {
+  font-family: 'Roboto', sans-serif;
+}
+
 .content {
+  font-family: 'Roboto Condensed', sans-serif;
   padding-top: 2vh;
   padding-left: 12vw;
   z-index:0;
 }
- 
-.section {
-  padding-top: 30vw;
-}
 
 #header {
-  padding-top: 8vh;
+  padding-top: 14vh;
 }
  
 /* ---- Top Navigation Bar ---- */
 #outer-nav {
-  background-color: #02003C;
-  color: #55FBDC;
   display: flex;
   flex-flow: row wrap;
+  font-family: 'Roboto Slab', serif;
   justify-content: space-between;
+  background-color:#41485B;
   padding: 2vmin;
   padding-top: 3vmin;
```

Comment:
can be combined as  padding: 3vmin 2vmin 2vmin;

Reply:
See #12 .